### PR TITLE
Meta

### PR DESCRIPTION
**Pull request by moses369**

### CODE REVIEW

**Comment by CountDown2Extinction** [comment]
test

**Comment by moses369** [comment]
hello

File: pages/_app.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
-import '../styles/globals.css'
-
+import "../styles/globals.css";
+import Meta from "../components/Meta";
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <>
```
Comment (JacobDWalters) [inline pages/_app.js:5]: why?

File: pages/index.js [hunk 2]
```diff
@@ -1,69 +1,11 @@
...
-        </a>
-      </footer>
-    </div>
-  )
+  useEffect(() => {
+    fetch("/api/hello")
+      .then((res) => res.json())
+      .then((data) => console.log(data));
+  }, []);
+  return <div className={styles.container}>Hello</div>;
```
Comment (EntropicWind) [inline pages/index.js:10]: does this work?

**Review by ilsaann — COMMENTED** [review]
is this a review?

**Review by JacobDWalters — APPROVED** [review]
looks great!

File: pages/index.js (hunk 2 above)
Comment (ilsaann) [reply pages/index.js:10]: what

**Review by EntropicWind — APPROVED** [review]

**Review by ilsaann — APPROVED** [review]

**Pull request merged**